go/internal/initutil/manager.go

```diff
@@ -166,6 +167,11 @@ func New(ctx context.Context) (*Manager, error) {
 		m.Datastore.defaultDir = storageDirs[0].Path
 	}
 
+	// InitialisaitonTimeout
+	if m.InitialisaitonTimeout == 0 {
+		m.InitialisaitonTimeout = time.Minute
+	}
```

Comment:
nope, if timeout is 0 just disable timeout checking in prepareForGetter

Reply:
> nope, if timeout is 0 just disable timeout checking in prepareForGetter

it is, that just for the default value, (I know that ~10 lines on top it says to put in CLI but I havn't made a CLI for that, do you think it's worth it ?)

Reply:
Yes, add a flag that sets it to 1 minute by default